addons/portal_rating/__manifest__.py

```diff
@@ -6,9 +6,12 @@
     'category': 'Hidden',
     'version': '1.0',
     'description': """
-Bridge module for rating on website stuff.
+Bridge module for rating on portal stuff.
```

Comment:
`name` above to be changed also to Portal Rating

Reply:
Bloup